collector/aspnet_aspnetapplications.go

```diff
@@ -0,0 +1,1239 @@
+// returns data points from Win32_PerfRawData_ASPNET_ASPNETApplications
```

Comment:
This will generate a linter error nowadays (wasn't the case when you initially submitted this PR). We now have the doc string along with the WMI class struct.